src/app.js

```diff
@@ -418,6 +418,15 @@ app.route("/schemas/*")
 app.route("/dist/*")
   .all(endpoints.static.auspiceAssets, (req, res, next) => next(new NotFound()));
 
+/* debug-narrative is a special page within auspice
+ * (currently experimental). It is not backed by a dataset,
+ * and only exists for GET requests for a HTML page
+ */
+app.route("/debug-narrative")
+  .get(async (req, res) => {
+    if (!req.accepts("text/html")) throw new NotAcceptable();
+    return await endpoints.static.sendAuspiceEntrypoint(req, res);
+  });
```

Comment:
To safely use async handlers, we have to use `routeAsync`, `getAsync`, etc. These are safe to use with non-async handlers too, so it's acceptable to always use them if you're unsure.

Furthermore, I would likely simplify this to:

```suggestion
app.routeAsync("/debug-narrative")
  .getAsync(endpoints.static.sendAuspiceEntrypoint);
```

It's ok for routes that don't support multiple content types to not bother with content negotiation, especially routes not intended for programmatic usage.

Reply:
Thanks!

If we don't do this, then were the `getAsync` to fail (reject) then it's unhandled because the middleware chain has already completed, whereas if we use `app.routeAsync` then the middleware will await the `getAsync` and so we can catch errors there?

Reply:
Yeah, errors can get lost/subvert the typical handling.